envoimoinscher.php

```diff
@@ -2473,6 +2708,7 @@ public function hookupdateCarrier($params)
 	 */
 	public function hookHeader()
 	{
+		$this->getContext()->controller->addJs('https://maps.google.com/maps/api/js?sensor=false');
```

Comment:
My bad, i thought we had to add the js/css in the header hook only, i'll change it now